src/dotnet/UI.Blazor/Services/BrowserInit/browser-init.ts

```diff
@@ -63,6 +63,10 @@ export class BrowserInit {
         }
     }
 
+    public static isStateOk() {
```

Comment:
can be expression method

Reply:
Looks like we do not use such approach in TS